src/bindings/python/src/pyopenvino/graph/passes/predicate.cpp

```diff
@@ -0,0 +1,125 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "pyopenvino/graph/passes/predicate.hpp"
+
+#include <pybind11/functional.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "openvino/pass/pattern/op/predicate.hpp"
+#include "pyopenvino/core/common.hpp"
+
+using Predicate = const ov::pass::pattern::op::ValuePredicate;
```

Comment:
do we actually use ValuePredicate in this file?

I think it's better to have such shortcuts:
using ValuePredicate = const ov::pass::pattern::op::ValuePredicate;
using Predicate = const ov::pass::pattern::op::Predicate;